### ISSUE TITLE
Missing nilpotency field for some Galois groups

### ISSUE THREAD

**Issue by edgarcosta** [LMFDB/lmfdb#4050]
https://beta.lmfdb.org/GaloisGroup/32T12882
https://beta.lmfdb.org/GaloisGroup/32T34907
https://beta.lmfdb.org/GaloisGroup/32T96908
```
Traceback (most recent call last):
  File "/home/sage/sage-9.1/local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/sage/sage-9.1/local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/sage/sage-9.1/local/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/sage/sage-9.1/local/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/sage/sage-9.1/local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/sage/sage-9.1/local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/lmfdb/lmfdb-git-dev/lmfdb/galois_groups/main.py", line 82, in by_label
    return render_group_webpage({'label': label})
  File "/home/lmfdb/lmfdb-git-dev/lmfdb/galois_groups/main.py", line 268, in render_group_webpage
    data['nilpotency'] = '$%s$' % data['nilpotency']
KeyError: 'nilpotency'
```

**Comment by jwj61** [comment]
This should be fixed on beta (and will be fixed on production when gps_transitive is uploaded).

I used upsert to add the missing data (there were 15 cases), but I still get

sage: db.gps_transitive.count({'nilpotency': {'$exists': false}})
15

However,

sage: foo=db.gps_transitive.search({'nilpotency': {'$exists': false}})
sage: foo=[z for z in foo]
sage: len(foo)
0

indicates that there really are 0 cases left.  Do I need to do something to update the counts table?  I tried  to call refresh_stats, but did not find the right way to call it.
